src/Appwrite/Platform/Workers/Messaging.php

```
@@ -466,6 +485,48 @@ private function getSmsAdapter(Document $provider): ?SMSAdapter
+    {
+        $credentials = $provider->getAttribute('credentials');
+
+        return match ($provider->getAttribute('provider')) {
+            'vonage-messages' => new VonageWhatsApp(
+                $credentials['applicationId'] ?? '',
+                $credentials['privateKey'] ?? '',
+                $credentials['from'] ?? null
+            ),
+            default => null
+        };
+    }
```

PHPMD (2.15.0)

[warning] Avoid unused private methods such as 'getWhatsAppAdapter'. (undefined)

(UnusedPrivateMethod)